### PR TITLE
Editorial updates 2022 08 08

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

**Review by kalxas — APPROVED** [review]

**Review by kalxas — APPROVED** [review]

File: workshop/content/docs/setup.md [hunk 1]
```diff
@@ -1,85 +1,86 @@
...
-[Docker.com](https://www.Docker.com/) remains the main entry.
+Docker has been available for almost 10 years, and provided as a deployment option on numerous FOSS software and OSGeo projects. Given the current
+era of computing, chances are that you have heard of Docker and `containerization`. Or, perhaps are already familiar and hopefully using Docker already.
+If not, there is a wide array of introductory materials that can be found online ([example from IBM](https://www.ibm.com/in-en/cloud/learn/Docker).
 
-Especially for the deployment and integration of Open Source Geospatial components, Docker
-is a blessing as it provides consistent packaging, isolation, integration and upgrade patterns, compared
-to custom installations. Though today we mainly use Docker, the bigger picture is the use of *Containers* as a next step
-in virtualization, but that would deserve its own workshop...
+FOSS4G software has benefitted greatly from Docker (consistent packaging, isolation, integration and upgrade patterns) in
```
Comment (justb4) [inline workshop/content/docs/setup.md:32]: I liked the "blessing" but ok, let's not bring in religion, so go ahead!

File: workshop/content/docs/setup.md [hunk 2]
```diff
@@ -1,85 +1,86 @@
...
+comparison to custom installations. Though today we mainly use Docker, the bigger picture is the use of **Containers** as a next step
+in virtualization. Containerization certainly deserves its own workshop, so for the purpose of this workshop we cover the basics
+of Docker and Docker Compose.
 
-[Docker Compose](https://docs.Docker.com/compose/) is an addition to Docker to facilitate 
-the orchestration (configuration) of one or more Docker 'Containers' (a Container is a running instance of a Docker image) 
-using a configuration convention, "The Docker Compose YAML File", usually named `Docker-compose.yml`.
+[Docker Compose](https://docs.docker.com/compose) is an addition to Docker to facilitate
```
Comment (justb4) [inline workshop/content/docs/setup.md:37]: good: you caught all the Capitalized instances that need lowercase!

File: workshop/docker/pygeoapi/docker.config.yml [hunk 3]
```diff
@@ -233,7 +233,7 @@ resources:
 #            - type: feature
```
Comment (justb4) [inline workshop/docker/pygeoapi/docker.config.yml:233]: Think we agreed to leave out all config sections, also the commented ones, except for the "First" slice.

File: workshop/html/vector-tiles.html [hunk 4]
```diff
@@ -9,7 +9,7 @@
     map = L.map('map').setView({ lat: 43.79, lng: 11.25 }, 12);
     map.addLayer(
         new L.tileLayer('https://stamen-tiles-{s}.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.{ext}', {
-        attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+        attribution: 'Map tiles by <a href="https://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
         subdomains: 'abcd',
```
Comment (justb4) [inline workshop/html/vector-tiles.html:13]: suggest to also add an OpenLayers example.

**Review by justb4 — APPROVED** [review]
Some minor comments, to do in next iteration/PR.

**Pull request merged**